build-config/make/busybox.make

```diff
@@ -4,6 +4,7 @@
 #  Copyright (C) 2014 david_yang <david_yang@accton.com>
 #  Copyright (C) 2014 Mandeep Sandhu <mandeep.sandhu@cyaninc.com>
 #  Copyright (C) 2014 Nikolay Shopik <shopik@inblock.ru>
+#  Copyright (C) 2016 Pankaj Bansal <pankajbansal3073@gmail.com>
 #
```

Comment:
I tried applying busybox 1.20.2 patches to busybox 1.25.0, but i am facing many compilation errors with this. i suggest that you please merge these changes to busybox 1.25.0 and update the busybox version.